docusaurus.config.js

```diff
@@ -75,4 +75,22 @@ module.exports = {
       },
     ],
   ],
+  plugins: [
+    [
+      require.resolve("@cmfcmf/docusaurus-search-local"),
+      {
+        indexDocs: true,
+        language: "th",
+        maxSearchResults: 10,
+        lunr: {
+          b: 0.75,
+          k1: 1.2,
+          titleBoost: 5,
+          contentBoost: 1,
+          tagsBoost: 3,
+          parentCategoriesBoost: 2,
```

Comment:
there's a plan to use algolia search, currently I'm still waiting for their reply: https://github.com/thedataengineeringbook/thedataengineeringbook#todo

results would look like something on this site: https://docs.karnwong.me/

cc @zkan

Reply:
Oh, I see that you applied to their program. I'm very thankful to you. 😍

For this PR, since this is the Hacktoberfest season and we can see the effort in it, how about we accept this PR and once you have their reply we can switch to Algolia DocSearch? We can create an issue, so that we can keep track it. 😊

Reply:
sounds good to me! let's keep the spirit going!